earn/src/data/MarginAccount.ts

```diff
@@ -309,224 +390,6 @@ export function priceToSqrtRatio(price: number, token0Decimals: number, token1De
     .mul(BIGQ96);
 }
 
-// function _computeProbePrices(sqrtMeanPriceX96: Big, sigma: number): [Big, Big] {
-//   sigma = Math.min(Math.max(ALOE_II_SIGMA_MIN, sigma), ALOE_II_SIGMA_MAX);
-//   sigma *= ALOE_II_SIGMA_SCALER;
-
-//   let a = sqrtMeanPriceX96.mul(new Big((1 - sigma) * 1e18).sqrt()).div(1e9);
-//   let b = sqrtMeanPriceX96.mul(new Big((1 + sigma) * 1e18).sqrt()).div(1e9);
-
-//   // Constrain to be within TickMath's MIN_TICK and MAX_TICK
-//   if (a.lt('4295128740')) {
-//     a = new Big('4295128740');
-//   }
-//   if (b.gt('1461446703485210103287273052203988822378723970341')) {
-//     b = new Big('1461446703485210103287273052203988822378723970341');
-//   }
-//   return [a, b];
-// }
```

Comment:
Is all this stuff being deleted because we don't need it for Earn?

Reply:
yes, I forgot to delete it earlier on.